proposals/PROPOSALS.md

```diff
@@ -0,0 +1,146 @@
+An HFTP (*Haskell Foundation Technical Proposal*) is a process for submitting a proposal for a technical initiative to be undertaken by the Haskell Foundation (HF). This document describes how the community may propose, discuss, and implement technical initiatives to be carried out through the Haskell Foundation that will affect the Haskell ecosystem. Each HFTP goes through an iterative process in which it is discussed by the HF Technical Track (HFTT), the broader community, and any and all interested stakeholders. If a consensus is reached, the initiative is accepted and may then be executed by the Foundation (resources permitting). Only upon reaching a consensus are initiatives accepted to be merged and executed. 
```

Comment:
What does "undertaken by the Haskell Foundation" mean? Are HF members going to execute the proposal once it is accepted? Or will the HF support the proposal's author in executing the proposal? It's important to clarify HF's role here.